source/crud/tests/v2/find-allowdiskuse-error.yml

```diff
@@ -0,0 +1,26 @@
+runOn:
+  - { maxServerVersion: "4.2" }
```

Comment:
Depending on whether this is still relevant in light of my earlier comment, I think prior art would suggest separate tests for client and server-side errors (with appropriate APM assertions), as in [deleteOne-hint-clientError.yml](https://github.com/mongodb/specifications/blob/master/source/crud/tests/v2/deleteOne-hint-clientError.yml) and 
[deleteOne-hint-serverError.yml](https://github.com/mongodb/specifications/blob/master/source/crud/tests/v2/deleteOne-hint-serverError.yml).

Reply:
done